LiteFarmOrg/LiteFarm
# Lf 2632 point locations are being inappropriately clustered

This PR resolves an issue where some points were being clustered when they shouldn't have been clustered.

To test:
1. Add two gates to your map, which are very close but not touching each other when the map is on max zoom
2. When you zoom out, they should be grouped into a marker
3. When you zoom in, they should separate out into two separate icons
